.travis.yml

```diff
@@ -12,8 +12,13 @@ addons:
       - libspatialite-dev
 
 python:
-  - "3.4"
-  - "2.7"
+  - "3.5"
```

Comment:
I think you should remove `Python 3.5` :smile:

Reply:
I was going to ask that whether it's needed, because if not I can enhance it a bit further, eg introducing f-strings :smile:

Reply:
Yes, we are currently moving to python 3.6 and + ! :smile: